modules/core/src/main/java/org/apache/ignite/configuration/CacheConfiguration.java

```diff
@@ -531,6 +519,11 @@ public CacheConfiguration(CompleteConfiguration<K, V> cfg) {
         writeBehindFlushSize = cc.getWriteBehindFlushSize();
         writeBehindFlushThreadCnt = cc.getWriteBehindFlushThreadCount();
         writeSync = cc.getWriteSynchronizationMode();
+        storeConcurrentLoadAllThreshold = cc.getStoreConcurrentLoadAllThreshold();
+        maxQryIterCnt = cc.getMaxQueryIteratorsCount();
+        sqlOnheapCache = cc.isSqlOnheapCacheEnabled();
+        sqlOnheapCacheMaxSize = cc.getSqlOnheapCacheMaxSize();
+        evtsDisabled = cc.isEventsDisabled();
```

Comment:
In the reverted commit, we made this list in alphabetical order. We could save this change.

Reply:
Yep, makes sense to me. Will do.